agent/wpthook/test_state.h

```diff
@@ -35,7 +35,8 @@ class DevTools;
 class Trace;
 
 const int TEST_RESULT_NO_ERROR = 0;
-const int TEST_RESULT_TIMEOUT = 99997;
+const int TEST_RESULT_OVERALL_TIMEOUT_REACHED_ERROR = 99995;

```

Comment:
I don't see why Pat had these errors arbitrarily at 9xxxx. But, we shouldn't make it more arbitrary by skipping 99996 and having 99995 instead unless there is a good reason.